lib/components/Keyboard/KeyboardInput/KeyboardAccessoryView.tsx

```diff
@@ -78,10 +67,10 @@ export type KeyboardAccessoryViewProps = kbTrackingViewProps & {
  * @gif: https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/KeyboardAccessoryView/KeyboardAccessoryView.gif?raw=true
  */
 class KeyboardAccessoryView extends Component<KeyboardAccessoryViewProps> {
-  static iosScrollBehaviors = IOS_SCROLL_BEHAVIORS;
+  // @ts-expect-error
+  static iosScrollBehaviors = KeyboardTrackingView.scrollBehaviors;
 
   static defaultProps = {
-    iOSScrollBehavior: -1,
```

Comment:
Why not change this to `KeyboardAccessoryView.iosScrollBehaviors.FIXED_OFFSET` instead of adding it bellow?

Reply:
Because I don't want to have a default to 'iOSScrollBehavior' prop so I can use the 'scrollBehavior' prop instead (that holds the default)